runtime-sdk/src/context.rs

```diff
@@ -289,7 +296,7 @@ impl<'a, R: runtime::Runtime, S: NestedStore> RuntimeBatchContext<'a, R, S> {
             key_manager,
             logger: get_logger("runtime-sdk")
                 .new(o!("ctx" => "dispatch", "mode" => Into::<&'static str>::into(&mode))),
-            block_tags: Tags::new(),
+            block_etags: BTreeMap::new(),
```

Comment:
Should this be `EventTags::new()` even thought the type is currently the same. Also below.